spec/controllers/admin/order_cycles_controller_spec.rb

```diff
@@ -323,13 +328,18 @@ def build_resource
           format: :json, id: order_cycle.id, order_cycle: allowed.merge(restricted)
         }
       }
-      let(:form_mock) { instance_double(OrderCycles::FormService, save: true) }
+      let(:form_mock) {
+        instance_double(OrderCycles::FormService, save: true)
+      }
 
       before { allow(controller).to receive(:spree_current_user) { user } }
 
       context "as a manager of the coordinator" do
         let(:user) { coordinator.owner }
-        let(:expected) { [order_cycle, allowed.merge(restricted), user] }
+        let(:error_class) { Admin::OrderCyclesController::DateTimeChangeError }
```

Comment:
I believe this is not used ?

Reply:
I needed to add `error_class` to match the mocked `OrderCycles::FormService` class as `@error_class` is now set in the initializer